index.bs

```diff
@@ -7797,10 +7797,12 @@ before any author-defined script have run.
 TODO: Extend this to scripts in other kinds of realms.
 
 A [=BiDi session=] has a <dfn>preload script map</dfn> which is a [=/map=] in
-which the keys are [[!RFC9562|UUID]]s, and the values are [=structs=] with an <a
-for=struct>item</a> named <code>function declaration</code>, which is a string,
-<code>arguments</code>, <code>contexts</code>, which is a list or null, and an item named <code>sandbox</code> which is a string
-or null.
+which the keys are [[!RFC9562|UUID]]s, and the values are [=structs=] with
+an [=struct/item=] named <code>function declaration</code>, which is a string,
+an [=struct/item=] named <code>arguments</code>, which is a list,
+an [=struct/item=] named <code>contexts</code>, which is a list or null,
+an [=struct/item=] named <code>sandbox</code>, which is a string or null,
+and an [=struct/item=] named <code>user contexts</code>, which is a [=/set=], initially empty.
```

Comment:
Not sure it makes sense to state that it's initially empty? For each preload script, the `user contexts` set will be assigned the value provided in the command. It's not like it starts empty and can be filled afterwards? We could say it's potentially empty, but not sure that's worth calling out?

Reply:
No preferences here, updated.